config/environments/production.rb

```diff
@@ -77,7 +78,7 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
-  config.active_job.queue_adapter = :sidekiq
+  config.active_job.queue_adapter = :inline # :sidekiq
 
-  # config.relative_url_root = "/pmp"
+  config.relative_url_root = "/scores"
```

Comment:
I'm still not sure what this value will be, but if we change this to 'pages' then it will be generic enough for IUPUI if they pull master, and that might be what we want to run as. If they want it to be 'po', then we can always change it during deployment.